pkgs/servers/home-assistant/frontend.nix

```diff
@@ -4,11 +4,11 @@ buildPythonPackage rec {
   # the frontend version corresponding to a specific home-assistant version can be found here
   # https://github.com/home-assistant/home-assistant/blob/master/homeassistant/components/frontend/manifest.json
   pname = "home-assistant-frontend";
-  version = "20200603.2";
+  version = "20200617.0";
```

Comment:
0.111.4 uses an older frontend version: https://github.com/home-assistant/core/blob/0.111.4/homeassistant/components/frontend/manifest.json#L5

Reply:
I have to fix my upgrade script...